imdown-mobile/squads/add_squad.tsx

```diff
@@ -70,7 +96,7 @@ const AddSquadModal = (props: OwnProps) => {
                 placeholderTextColor = "lightgray"/>
             </SafeAreaView>
             
-            <TouchableOpacity onPress={addSquad} style={AddSquadStyles.add_squad_button}>
+            <TouchableOpacity onPress={onPress} style={AddSquadStyles.add_squad_button}>
```

Comment:
Eh I feel like there doesn't need to be a separate onPress method if all its doing is calling `addSquadOnBackend()`, might as well just put `addSquadOnBackend` here